.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: false
+
+language: node_js
+
+node_js: 8.9.3
+
+cache:
+  directories:
+    - $HOME/.npm
+
+install:
+  - npx "npm@$NPM_VERSION" ci
+
+branches:
+  only:
+    - master
+
+script: npx "npm@$NPM_VERSION" run ci
```

Comment:
Ooh you could add integration tests for this module by making it publish a comment on each of its own PRs 😄

Reply:
;)